Plugins/Gradle/src/main/kotlin/dev/testify/tasks/utility/SettingsTask.kt

```diff
@@ -81,10 +117,16 @@ open class SettingsTask : TestifyUtilityTask() {
 
     override fun taskAction() {
         val userId = Adb.forcedUser?.toString() ?: Device.user.takeUnless { Device.isEmpty } ?: "Device not found"
+        val notSet =
+            "${AnsiFormat.Bold}${AnsiFormat.Green}${project.gradle.startParameter.consoleOutput}${AnsiFormat.Reset}"
```

Comment:
Using `project` in here breaks the gradle configuration cache

Reply:
Yeah, I think it makes sense to just not expose this in settings at all since it isn't unique to the plugin.